arangod/Statistics/StatisticsWorker.cpp

```diff
@@ -1049,9 +1049,8 @@ void StatisticsWorker::run() {
       LOG_TOPIC(WARN, Logger::STATISTICS) << "caught unknown exception in StatisticsWorker";
     }
 
-    seconds++;
+    seconds++; // TODO should this run adaptively ?
+    std::this_thread::sleep_for(std::chrono::seconds(1));
 
-    CONDITION_LOCKER(guard, _cv);
-    guard.wait(1000 * 1000);
```

Comment:
Why has the condition locker been removed here? I think it was used to signal server shutdown so the statistics thread could finish early when waiting in the `wait` call. With the `wait_for`, that functionality has now been removed.
Intentionally?